confdb/aclocal_util.m4

```diff
@@ -58,17 +58,36 @@ AC_DEFUN([PAC_PREFIX_ALL_FLAGS],[
 	PAC_PREFIX_FLAG($1, EXTRA_LIBS)
 ])
 
+AC_DEFUN([PAC_FGREP_WORD],[
+	AC_REQUIRE([AC_PROG_FGREP])
+        AC_CACHE_CHECK([check if fgrep support -w option], ac_cv_path_FGREP_has_word,
+           [if echo 'ab*c' | $GREP -Fw 'ab*c' >/dev/null 2>&1
+           then ac_cv_path_FGREP_has_word="yes"
+           else ac_cv_path_FGREP_has_word="no"
+           fi])
+        if test x$ac_cv_path_FGREP_has_word = "xyes"; then
+            AS_IF(
+                    [echo "$$2" | $FGREP -ew "$1" >/dev/null 2>&1],
+                    [$3],
+                    [$4]
+            )
+        else
+            AC_WARNING([grep does not support -w option, skip flag deduplication])
```

Comment:
How about moving the warning into `AC_CACHE_CHECK` so we don't end up with dozens of the same warning?

Reply:
Done. And yes manual test with CUDA is working correctly. So does the gpu-ucx job.